nrf_security/src/CMakeLists.txt

```diff
@@ -7,12 +7,14 @@
 #
 # File including instructions to build PSA APIs.
 #
-
-set(OBERON_PSA_PATH ${NRFXLIB_DIR}/crypto/nrf_oberon)
-
+# Can be executed by both the NCS and TF-M build systems.
 #
+# For TF-M the nrf_security/tfm/CMakeLists.txt file will execute this file.
+# For NCS  the nrf_security/CMakeLists.txt     file will execute this file.
 #
 
+set(OBERON_PSA_PATH ${NRFXLIB_DIR}/crypto/nrf_oberon)
```

Comment:
The variable is set here as well but not the (helpful) comments above. Clean that up please. 
https://github.com/nrfconnect/sdk-nrfxlib/pull/1010/commits/4920b1088c8c772656a23e05a38c52599f92f440